tests/mock_gui/app.py

```diff
@@ -0,0 +1,13 @@
+import pyblish.cli
```

Comment:
I'd rather we make these written from within the test, then hosted as individual files.

I've been bitten by this once before, the problem arises when we update Pyblish and this interface. We'll still want to test for the old behaviour, for backwards compatibility, but also test for the new one. So then you'll need to duplicate these files.

See the [`preXX`](https://github.com/pyblish/pyblish-base/tree/master/tests) folders I had to do to combat this..

If you write them into a temporary folder in the `setup()` of the test module, it'll run once before all tests. Then in `teardown()` you can `shutil.rmtree` it.